curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-through-an-array-with-a-for-loop.english.md

```diff
@@ -76,13 +72,11 @@ var myArr = [ 2, 3, 4, 5, 6];
 ```
 
 </div>
-
```

Comment:
@moT01 This extra line will need to be in there.  See my PR https://github.com/freeCodeCamp/freeCodeCamp/pull/36278
```suggestion

```